### PR TITLE
[ci] Tweak scripts

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Branch? | 2.3 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | https://github.com/sebastianbergmann/phpunit-mock-objects/issues/299 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by stof** [comment]
@nicolas-grekas please make ``./phpunit install`` exit with a failure exit code when composer fails, to avoid the weird build issues we got here

**Comment by nicolas-grekas** [comment]
@stof good idea, done (and workaround reverted thx to v3.1.1)

File: appveyor.yml [hunk 1]
```diff
@@ -38,10 +38,8 @@ install:
     - IF %PHP%==1 echo max_execution_time=1200 >> php.ini-min
     - IF %PHP%==1 echo date.timezone="UTC" >> php.ini-min
     - IF %PHP%==1 echo extension_dir=ext >> php.ini-min
-    - IF %PHP%==1 echo extension=php_openssl.dll >> php.ini-min
-    - IF %PHP%==1 echo extension=php_curl.dll >> php.ini-min
-    - IF %PHP%==1 echo curl.cainfo=c:\php\cacert.pem >> php.ini-min
     - IF %PHP%==1 copy /Y php.ini-min php.ini-max
+    - IF %PHP%==1 echo extension=php_openssl.dll >> php.ini-max
```
Comment (stof) [inline appveyor.yml]: are you sure about it ? running composer requires openssl

Comment (nicolas-grekas) [inline appveyor.yml]: yes sure, thx to https://github.com/symfony/symfony/pull/18276/files/82c6bed6b0f7f3f66ad0f98ae40c35dcad328575#diff-180360612c6b8c4ed830919bbb4dd459L53

Comment (stof) [inline appveyor.yml]: OK, just saw that composer always runs with the max ini file so it is OK

**Pull request merged**